recipe/meta.yaml

```
@@ -26,14 +27,15 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ stdlib('c') }}
     - patch
     - make
   host:
     - python
     - pip
-    - setuptools
+    - setuptools >=59.0
     - wheel
-    - cython >=3
+    - cython >=3,<4
     # These are required for the vendored htslib.
     - libcurl 8
```

We've gotten a lot better at declaring the minimum versions of packages we can use in the `host` section as Jinja variables.  Here, they are defined in `aggregate`'s `cbc.yaml`: for example `libcurl` https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L427

If such an entry exists in `cbc.yaml` you can replace the entry in the `host` section such as:

```suggestion
    - libcurl {{ libcurl }}
```

so the same for `libdeflate` and `lz4-c`